ror/app/controllers/articles_controller.rb

```diff
@@ -45,6 +49,6 @@ def destroy
 
   private
     def article_params
-      params.require(:article).permit(:title, :body, :status)
+      params.permit(:title, :body, :status)
```

Comment:
Fixed. Some notes for future: when posting requests via Postman, it is necessary to nest the request parameters like "article[title]", "article[body]". Otherwise, rails will throw "ActionController::ParameterMissing" exception.